### DISCUSSION TITLE
Do Wii U's Mii.py still supporting on Nintendo network?

### DISCUSSION THREAD

**Discussion by Freshestocto** [Kinnay/NintendoClients#180] [General]
Bcs it used to working (first pic) but after the Nintendo network down it will only display an error now (second pic)
I'm pretty curious is it still working or it's no longer supporting?
![Screenshot_2024-10-13-18-45-17-58_572064f74bd5f9fa804b05334aa4f912](https://github.com/user-attachments/assets/564c9763-c05c-4050-affd-15ccaa6a76ed)
![Screenshot_2024-10-13-18-44-28-63_572064f74bd5f9fa804b05334aa4f912](https://github.com/user-attachments/assets/581de73a-c6cf-453d-91ca-0ada543390e9)

**Comment by kinnay** [comment]
Hmm you are right. I don't think it's possible to request miis anymore since the network got shut down.

That raises the question whether I want to keep the 3DS / Wii U implementations in this repository. Maybe I will remove them from the main branch at some point.

**Comment by jonbarrow** [comment]
> since the network got shut down

From what we've seen over at Pretendo the Mii images endpoint is the only one which was actively taken down. Every other endpoint still functions as normal, so I wouldn't necessarily call it "shut down" right now to be honest

> That raises the question whether I want to keep the 3DS / Wii U implementations in this repository. Maybe I will remove them from the main branch at some point

This feels a *bit* nuclear imo? A number of servers are still online (the friends server and Pokemon Bank), so there's still use for it. On top of that, projects like https://github.com/MCMi460/3DS-RPC use this client for interacting with 3DS servers, including ours, so there's still people actively using the Wii U/3DS support in the client. Removing it entirely seems like a net-negative to me?

**Comment by Freshestocto** [comment]
> Hmm you are right. I don't think it's possible to request miis anymore since the network got shut down.
> 
> That raises the question whether I want to keep the 3DS / Wii U implementations in this repository. Maybe I will remove them from the main branch at some point.

I see : o
